dashboard/app/controllers/aichat_controller.rb

```diff
@@ -1,4 +1,6 @@
 ROLES_FOR_MODEL = %w(assistant user).freeze
+DEFAULT_POLLING_INTERVAL_MS = 1000
```

Comment:
Kind of roughly - through testing we've seen that SageMaker responses can take anywhere from <1s to 20+s. We don't want to poll any faster than once per second to avoid overloading our backend, but we also want the endpoint to be responsive if the response comes back really quickly. The backoff rate also allow us to slow down the rate of requests if we feel like it's happening too fast.

Reply:
Gotcha - thanks for the explanation!